src/components/dialogs/generator-modification-dialog.js

```diff
@@ -283,8 +266,8 @@ const GeneratorModificationDialog = ({
                 reactivePowerSetpoint,
                 voltageRegulation,
                 voltageSetpoint,
-                connectivity?.voltageLevel?.id,
-                connectivity?.busOrBusbarSection?.id,
+                undefined,
```

Comment:
Does it overwrite the Generator already existing `connectivity?.voltageLevel?.id` and `connectivity?.busOrBusbarSection?.id` in the back or is it ignored ?